toc/working-groups/app-runtime-interfaces.md

```diff
@@ -300,9 +300,7 @@ areas:
   - cloudfoundry/staticfile-buildpack-release
 
 - name: OSB API
-  approvers:
-  - name: Rodrigo Sampaio Vaz
-    github: rsampaio
+  approvers: []
```

Comment:
If we remove @rsampaio, then this area will have no remaining approvers.

This area was just added to ARI in https://github.com/cloudfoundry/community/commit/3e124fed3ccdca3451527cd4bda998702ff6cccf.

If we can't find new approvers, should we fold the repos into CAPI?